automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -71,6 +74,19 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     serverExtensions = Map("jupyterlab" -> "jupyterlab")
   )
 
+  // TODO: move this to NotebookTestUtils and chance cluster-specific functions to only call if necessary after implementing RStudio
+  def saveJupyterLogFile(clusterName: ClusterName, googleProject: GoogleProject, suffix: String)(implicit token: AuthToken): Try[File] = {
+    Try {
+      val jupyterLogOpt = notebooks.Notebook.getContentItem(googleProject, clusterName, "jupyter.log", includeContent = true)
```

Comment:
Instead of `notebooks.Notebook...` can we import `import org.broadinstitute.dsde.workbench.leonardo.notebooks.Notebook`  and the use `Notebook.getContentItem(...)`? It would make it easier to understand and navigate the code.